Sources/BraintreeDropIn/BTPaymentSelectionViewController.m

```diff
@@ -226,7 +227,7 @@ - (void)configurationLoaded:(__unused BTConfiguration *)configuration error:(NSE
     NSMutableArray *activePaymentOptions = [@[] mutableCopy];
     if (!error) {
         [self fetchPaymentMethodsOnCompletion:^{
-            if ([[BTTokenizationService sharedService] isTypeAvailable:@"PayPal"] && [self.configuration.json[@"paypalEnabled"] isTrue] && !self.dropInRequest.paypalDisabled) {
```

Comment:
The PayPal module is required for Drop-in, so we don't need to check that it's available.